tools/mason/MasonSearch.chpl

```diff
@@ -98,8 +97,12 @@ proc masonSearch(ref args: list(string)) {
       }
     }
   }
-  var res = rankResults(results, query);
-  for r in res do writeln(r);
+  var res: list(string);
+  res = rankResults(results, query);
```

Comment:
Can we not leave this as:

```suggestion
  var res = rankResults(results, query);
```

Reply:
I think the space between package and version is intentional for readability (`A(0.1.0)` vs. `A (0.1.0)`). Also, this code might be more readable by being a little clearer about the print formatting, e.g.

```chpl
for package in res {
  const (name, version) = package.split('.');
  writef('%s (%s)\n', name, version.replace('_', '.'));
}